plugin/metadata.go

```diff
@@ -0,0 +1,58 @@
+package plugin
+
+import "errors"
+
+// Metadata provided by the plugin.
+type Metadata struct {
+	Name                      string   `json:"name"`
+	Description               string   `json:"description"`
+	Version                   string   `json:"version"`
+	URL                       string   `json:"url"`
+	SupportedContractVersions []string `json:"supported-contract-versions"`
```

Comment:
>supported-contract-versions

Should it be `supportedContractVersions`?

Reply:
Good catch!